tasks/main.yml

```diff
@@ -1,33 +1,34 @@
 ---
-- name: Load var file with package names based on the OS type
+- name: Load var file with package names based on the package manager
   ansible.builtin.include_vars:
     file: "{{ lookup('first_found', params) }}"
   vars:
     params:
       files:
-        - "{{ ansible_distribution }}_{{ ansible_distribution_release }}.yml"
-        - "{{ ansible_distribution }}.yml"
-        - "{{ ansible_os_family }}.yml"
+        - "{{ ansible_pkg_mgr }}.yml"
```

Comment:
Would it make sense to preemptively use the same configuration as the `include_tasks` below?

Reply:
Please see commit [d839ff9](https://github.com/cisagov/ansible-role-automated-security-updates/pull/38/commits/d839ff9f21355e9fe741ae79e50b6825ff00fac8).